src/aurelia-slickgrid/custom-elements/aurelia-slickgrid.ts

```diff
@@ -108,13 +99,26 @@ export class AureliaSlickgridCustomElement {
   private _isLocalGrid = true;
   private _paginationOptions: Pagination | undefined;
   private _registeredResources: ExternalResource[] = [];
+  private _columnDefinitionObserver: ICollectionObserver<CollectionKind.array>;
+  private _columnDefinitionsSubscriber: ICollectionSubscriber = {
+    handleCollectionChange: () => {
```

Comment:
@MaximBalaganskiy 
I'm back to working on this migration and that this `_columnDefinitionObserver` only works once and not after, would you have any clue why that is? 

to replicate the issue, you can get latest code and go to Example 3 then click on the "Dynamically Duplicate Title Column" button multiple times, it only add an extra column on the first time but not after and after debugging and adding a console log in here, I see that the subscriber only worked once